.codeclimate.yml

```diff
@@ -28,9 +28,3 @@ plugins:
   rubocop:
     enabled: true
     config: ".rubocop_cc.yml"
```

Comment:
You'll need `channel: rubocop-0-69` here to fix the codeclimate failure but we can do that in a follow-up PR